linkerd/proxy/transport/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "linkerd2-proxy-transport"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+description = """
+Transport-level implementations that rely on core proxy infrastructure
+
+This should probably be decomposed into smaller, decoupled crates.
```

Comment:
Again, I think it would be good to know if this crate is actually a bottleneck in the compilation process or not...

Reply:
Again, it's less about compile time and more about logical units. There are smaller units in this module that should be made logically independent.